app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,65 @@
+.divider {
+  margin: 1.6rem;
+}
+
+.form-divider {
+  margin: 0;
+}
+
+.info-description {
+  border-top: 2px dotted $primary-color;
+  margin-top: 0.3rem;
+  padding-top: 0.3rem;
+  font-size: 0.8rem;
+}
+
+.info-name {
+  float: left;
+  color: $primary-color;
+  font-weight: bold;
+  font-size: 1.1rem;
+}
+
+.info-price {
+  float: right;
+  font-weight: bold;
+  margin-right: 0.5rem;
+}
+
+.info-quantity {
+  float: right;
+
+  [type="text"],
+  textarea {
+    width: 2rem;
+    height: 1.75rem;
+    margin-top: -0.2rem;
+    margin-right: 0;
+    margin-bottom: 0;
+    padding: 0;
+  }
+}
+
+.menu-title {
+  text-align: center;
+}
+
+.block-list ul {
+  margin: 1em;
+  border: 1px solid $secondary-color;
+  font-size: 0.875rem;
+  line-height: 1.6;
+  list-style-position: outside;
+  margin-bottom: 15px;
+}
+
+.block-list ul li {
```

Comment:
Merge rule `.block-list ul li` with rule on line 47<br>Selector should have depth of applicability no greater than 2, but was 3